### ISSUE TITLE
not working drag..

### ISSUE THREAD

**Issue by cjstk7168** [phphe/vue-draggable-nested-tree#90]
```
Uncaught TypeError: Cannot read property 'offsetHeight' of undefined
    at Object.drag (vue-draggable-nested-tree.es.js?d30b:1355)
    at drag (draggable-helper.esm.js?acb5:122)
    at HTMLDocument.moving (draggable-helper.esm.js?acb5:181)
    at HTMLDocument.wrapper (drag-event-service.es.js?6d58:79)
```

There is an error on this line..
**vue-draggable-nested-tree.es.js line number 1355**
`dplh.innerStyle.height = store.el.offsetHeight + 'px';` 

why not found `offsetHeight`?

<img width="511" alt="스크린샷 2019-08-12 오후 12 51 06" src="https://user-images.githubusercontent.com/39257313/62844959-e10e3d80-bcff-11e9-8292-e060d8541edd.png">

**Comment by phphe** [comment]
Ignore it if it does not affect the result. I am reconstructing this repo so I am not going to solve the issue.

**Comment by cjstk7168** [comment]
@phphe 
`draggable-helper` updated 2days ago.

**Comment by phphe** [comment]
fixed in 2.2.18
